spec/system/candidate_interface/candidate_reviewing_application_spec.rb

```diff
@@ -4,9 +4,28 @@
   include CandidateHelper
 
   scenario 'Candidate with all sections filled in' do
-    given_i_have_completed_my_application
-
-    when_i_click_on_check_your_answers
+    given_courses_exist
+    given_i_am_signed_in
+
+    %i[
+      course_choices
+      personal_details
+      contact_details
+      work_experience
+      volunteering
+      degrees
+      maths_gcse
+      english_gcse
+      science_gcse
+      becoming_a_teacher
+      subject_knowledge
+      interview_preferences
+      references
+    ].map { |section| check_and_fill_in_section_for(section) }
```

Comment:
Really like the thorough testing but this section here is a bit jarring when reading the scenario. Could we extract a method for this?

Reply:
Discussed offline, this is OK as the trade-off to doing that is that the fields get somewhat obfuscated.